packages/core/content-releases/admin/src/index.ts

```diff
@@ -15,11 +15,11 @@ const admin: Plugin.Config.AdminInput = {
         icon: PaperPlane,
         intlLabel: {
           id: `${pluginId}.plugin.name`,
-          defaultMessage: 'Releases',
+          defaultMessage: 'Content Releases',
         },
         async Component() {
           const { Releases } = await import(
-            /* webpackChunkName: "content-type-builder" */ './pages/Releases'
+            /* webpackChunkName: "content-type-builder" */ './pages/App'
```

Comment:
@joshuaellis with the recent tooling changes, can you confirm if chunk names still work?

Reply:
you can omit them, they'll be removed when we add vite anyway